public/model/region-details.js

```diff
@@ -43,11 +45,23 @@ var RegionDetails = P({
     return this.region_feature_collection.features;
   },
 
+  get_region_properties: function(region_code) {
+    return this.region_data_by_code[region_code];
+  },
+
   get_selected_regions_data: function() {
     var region_data_by_code = this.region_data_by_code;
     return this.selected_regions.get_region_codes().map(function(region_code) {
       return region_data_by_code[region_code];
     });
+  },
+
+  get_epi_data_display_string: function(date, region_code) {
```

Comment:
so, I see you're being clever and reusing `case_data_to_html_string` in Leaflet and also the details popup ... do you think it's worth using `dangerouslySetInnerHTML`?